torchgeo/datasets/sen12ms.py

```diff
@@ -62,13 +63,63 @@ class SEN12MS(VisionDataset):
        This download will likely take several hours.
     """  # noqa: E501
 
-    BAND_SETS: Dict[str, List[int]] = {
-        "all": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14],
-        "s1": [0, 1],
-        "s2-all": [2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14],
-        "s2-reduced": [3, 4, 5, 9, 12, 13],
+    BAND_SETS: Dict[str, Sequence[str]] = {
```

Comment:
It's fine to use `Tuple` here because this is obviously a tuple. But the function should still take `Sequence` because either `List` or `Tuple` can be used.
```suggestion
    BAND_SETS: Dict[str, Tuple[str, ...]] = {
```